src/types.ts

```diff
@@ -278,3 +287,25 @@ export interface BalanceUpdate {
   deltaWei: Integer;
   newPar: Integer;
 }
+
+// ============ Limit Orders ============
+
+export interface LimitOrder {
+  makerMarket: Integer;
+  takerMarket: Integer;
+  makerAmount: Integer;
+  takerAmount: Integer;
+  makerAccountOwner: address;
+  makerAccountNumber: Integer;
+  takerAccountOwner: address;
+  takerAccountNumber: Integer;
+  expiration: Integer;
+  salt: Integer;
+  signature?: string;
```

Comment:
can you instead make another type that is a `SignedLimitOrder` that extends `LimitOrder` and has the signature field? That would help with the typing in external clients too

Reply:
OK